lib/alchemy/admin/locale.rb

```diff
@@ -46,15 +46,16 @@ def available_locale
       def locale_from_user
         return if !current_alchemy_user
         if user_has_preferred_language?
-          current_alchemy_user.language
+          current_alchemy_user.language.to_sym
```

Comment:
Why call `to_sym` here, if we already call it at https://github.com/AlchemyCMS/alchemy_cms/pull/1098/files#diff-47a249f6fcd382200bbd5a7c64722b74L42?